Modules/_fuzz/fuzzer.cpp

```diff
@@ -0,0 +1,118 @@
+// A fuzz test for CPython.
+//
+// Unusually for CPython, this is written in C++ for the benefit of linking with
```

Comment:
"Unusually"?  This would be literally the first C++ code shipped by CPython that ran inside the interpreter.  (The existing CPP files shipped by CPython are all for Windows support, and all run *outside* the interpreter.)  It's my understanding that CPython has zero C++ in it very much on purpose.  I understand why you use C++ here, but I'd want to get BDFL blessing before I merged this file.

Also, a quick perusal of the documentation for libfuzzer suggests a strong dependency on Clang.  CPython supports at least two other compilers--gcc and Microsoft Visual C++--and supports at least one major platform where Clang is not a supported compiler (Windows).  Can you confirm that the build process and test suite will degrade gracefully when compiling with gcc or MSVC++?